packages/raccoon-city-ui/.env

```diff
@@ -1,5 +1,5 @@
-REACT_APP_GRAPHQL_URL="http://localhost:4000/graphql"
-REACT_APP_GRAPHQL_URL_="http://api-chessgrid.developerit.systems/graphql"
+REACT_APP_GRAPHQL_URL_="http://localhost:4000/graphql"
+REACT_APP_GRAPHQL_URL="http://api-chessgrid.developerit.systems/graphql"
 REACT_APP_URL="http://localhost:4000"
```

Comment:
NO!

Reply:
Setup prod db locally instead 
[raccoonCityDump.zip](https://github.com/didevgen/raccoon-city/files/5517809/raccoonCityDump.zip)

Reply:
got it